### PR TITLE
EZP-28745: Disabled scrolling to top of the page on requesting more s…

### PR DESCRIPTION
**Pull request by mateuszbieniek**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-28745](https://jira.ez.no/browse/EZP-28745)
| **Bug**| yes
| **Target version** | 1.7, 1.13

This PR fixes unwanted scrolling to the top of the page when requesting more results in Search View.

### CODE REVIEW

**Comment by mateuszbieniek** [comment]
Note: Travis check fails because of reasons unrelated to this PR.

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 1]
```diff
@@ -198,6 +198,9 @@ YUI.add('ez-platformuiapp', function (Y) {
                     oldService.setNextViewServiceParameters(newService);
                 }
             });
+
+            // Saving default value of scrollToTop for searchview to use.
+            this.set('scrollToTopDefault', this.get('scrollToTop'));
```
Comment (sunpietro) [inline Resources/public/js/apps/ez-platformuiapp.js]: I believe you should do it differently. You should set a default value in the attribute (in `ATTRS`) with read-only permission:
```javascript
scrollToTopDefault: {
  value: true,
  readOnly: true,
}
```
Then this line, you added, will be not needed.

File: Resources/public/js/apps/ez-platformuiapp.js [hunk 2]
```diff
@@ -1206,6 +1206,18 @@ YUI.add('ez-platformuiapp', function (Y) {
...
+            /**
+             * Boolean that determine if by default document should be scrolled to top on navigation
+             *
+             * @attribute scrollToTopDefault
+             * @default true
+             * @readOnly
+             * @type {Boolean}
+             */
+            scrollToTopDefault: {
+                value: true,
```
[the remark this answers is not in the document]
Comment (mateuszbieniek) [reply Resources/public/js/apps/ez-platformuiapp.js]: fixed in https://github.com/ezsystems/PlatformUIBundle/pull/979/commits/3040e8420469dbf133582040498848d2a221862a, pinging @sunpietro

**Comment by mateuszbieniek** [comment]
ping @micszo

**Comment by sunpietro** [comment]
ping @mnocon

**Comment by lserwatka** [comment]
@mateuszbieniek you can merge it up

**Comment by mateuszbieniek** [comment]
@lserwatka done

**Pull request merged**